policyengine_us/tests/policy/baseline/gov/states/az/tax/income/credits/az_family_tax_credit.yaml

```diff
@@ -0,0 +1,59 @@
+- name: AZ credit unit test 1
```

Comment:
moving forward, the test cases should have a name that explains what edge case we are testing